src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -135,15 +138,63 @@ public static Set<Tag> parseTags(Collection<String> tags) throws ParseException
      *
      * @throws ParseException if the given {@code tutorial} is invalid.
      */
-    public static Tutorial parseTutorial(String tutorial) throws ParseException {
+    private static Tutorial parseTutorial(String tutorial) throws ParseException {
         requireNonNull(tutorial);
         String trimmedTutorial = tutorial.trim();
+        try {
+            Integer.parseInt(trimmedTutorial);
+        } catch (NumberFormatException e) {
+            throw new ParseException(String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT,
+                    Tutorial.MESSAGE_INVALID_FORMAT));
+        }
         if (!Tutorial.isValidTutorial(trimmedTutorial)) {
             throw new ParseException(Tutorial.MESSAGE_CONSTRAINTS);
         }
         return new Tutorial(trimmedTutorial);
     }
 
+    /**
+     * Parses a {@code String tutorialInput} into a list of {@code Tutorial} objects.
+     * This method supports both single tutorials (e.g., "3") and multiple tutorials, either comma-separated
+     * (e.g., "[3,4,5]") or ranges (e.g., "3-5").
+     *
+     * @param tutorialInput the string representing multiple tutorials or a range
+     * @return a list of parsed {@code Tutorial} objects
+     * @throws ParseException if the input format is invalid
+     */
+    public static List<Tutorial> parseTutorials(String tutorialInput) throws ParseException {
+        requireNonNull(tutorialInput);
+        List<Tutorial> tutorials = new ArrayList<>();
+
+        // Check if it's a list of tutorials within brackets [3,4,5]
+        if (tutorialInput.startsWith("[") && tutorialInput.endsWith("]")) {
+            String content = tutorialInput.substring(1, tutorialInput.length() - 1).trim();
+            String[] parts = content.split(",");
+            for (String part : parts) {
+                tutorials.add(parseTutorial(part.trim()));
+            }
+        } else if (tutorialInput.contains("-")) {
+            String[] range = tutorialInput.split("-");
+            if (range.length != 2) {
+                throw new ParseException(String.format(Messages.MESSAGE_INVALID_COMMAND_FORMAT,
+                        Tutorial.MESSAGE_INVALID_FORMAT));
+            }
```

Comment:
Same comments as above, maybe we could use isValidTutorial here instead of doing parseInt ourselves. Perhaps can consider having a method in Tutorial that checks for start > end too instead of letting ParserUtil handle this directly.